pyflakes/checker.py

```diff
@@ -870,7 +870,16 @@ def handleNode(self, node, parent):
 
     def handleDoctests(self, node):
         try:
-            (docstring, node_lineno) = self.getDocstring(node.body[0])
+            if hasattr(node, 'docstring'):
+                docstring = node.docstring
+
+                # This is just a reasonable guess. In Python 3.7, docstrings no
```

Comment:
You can get better guess if inspect the maximal lineno of arguments, default values, annotations, etc. This could allow you to pass test_offsetWithMultiLineArgs in 3.7.